pyphare/pyphare/pharesee/run.py

```diff
@@ -89,6 +89,30 @@ def _compute_current(patch):
                 {"name":"Jy", "data":Jy, "centering":centering["Jy"]},
                 {"name":"Jz", "data":Jz,"centering":centering["Jz"]})
 
+
+def _divB2D(Bx,By, xBx, yBy):
+    dxbx = (Bx[1:,:]-Bx[:-1,:])/(xBx[1]-xBx[0])
```

Comment:
I'm never very happy with calculating a dx as the diff of value [0] and [1]. as a patch is defined on a grid, why dont we have a dedicated data like a tuple `(dx, dy, dz)` ?